Header.php

```diff
@@ -27,10 +27,11 @@
 <header>
  
    
-<link href="css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
+<link href="http://118.25.96.118/nor/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
```

Comment:
@dzr201732120115 

Why add "http://118.25.96.118/nor/"?  I think this would not make any difference.

**Hui**